Makefile.frankfurt

```diff
@@ -136,6 +137,32 @@ serve:  guard-DEPLOY_TARGET guard-OPENTRANS_API_KEY guard-PGUSER guard-PGPASSWOR
 shell:
 	PYTHONPATH=${PYTHONPATH} ${PSHELL} local.ini
 
+	.PHONY: dockerlogin
```

Comment:
`dockerlogin`, `dockerbuild` and `dockerrun` should also appear in the `help` target of this Makefile